ward/cparser.lua

```diff
@@ -138,7 +138,7 @@ local keywords = {
   'int', 'switch', 'double', 'long', 'typedef', 'else', 'register',
   'union', '__asm', '__asm__', '__thread', '__builtin_va_list', '__inline__',
   '__builtin_va_arg', '__extension__', '__const', '__restrict', '__volatile__',
-  '__signed', '__unsigned', '_Alignof',
+  '__signed', '__unsigned', '_Alignof', '_Nonnull',
```

Comment:
So, why is this handled here, but e.g. `_Noreturn` is handled in `config.lua` like this:
```
preprocessor_options = {
  "-D__extension__=",
  "-D__attribute__(x)=",
  -- Ignore C11 annotations.
  "-D_Noreturn=",
  "-D_Alignas(x)=",
  -- TODO: The following is a workaround around __typeof() occurring
  -- in Linux header files. The correct solution is to properly deal
  -- with typeof() and __typeof() expressions.
  "-D__typeof(x)=int"
}
```